scripts/display_results.py

```diff
@@ -4,20 +4,46 @@
 
 from markermap.utils import plot_benchmarks
 
+def mergeFiles(infiles, outfile):
+  datas = []
+  for file in infiles:
+    datas.append(np.load(file, allow_pickle=True).item())
+
+  data_1 = datas[0]
+  for i in range(1,len(datas)):
+    data_2 = datas[i]
+
+    for key1 in ['misclass', 'f1']:
+      for key2, arr in data_2[key1].items():
+        data_1[key1][key2] = np.concatenate((data_1[key1][key2], arr))
+
+  np.save(outfile, data_1)
+
```

Comment:
I am a bit confused about this function, but it makes sense. I think my confusion comes from not knowing the structure of the files.

Reply:
Yeah, this function was to help recombine files because I split the larger data sets into multiple runs of the script in order to parallelize the process because it was taking forever. Now that I am thinking about it again, I think this merge_files function doesn't really need to be in the repo, I will remove and just leave it outside the repo on my own machine.